### PR TITLE
feat(gangway): migration to gabibbo97 charts

### PR DESCRIPTION
**Pull request by eyenx**
# Description

Migrates the gangway chart to gabibbo97 charts, which are actively mantained.

# Issues

get rid of `charts.helm.sh/stable`

# Checklist

* [x] I updated the version in Chart.yaml
* [x] I updated applicable README.md files using  `pre-commit run -a`
* [x] I documented any high-level concepts I'm introducing in `docs/`
* [x] If I updated a dependency tool, or app, this PR contains a short summary of the changes I'm pulling
* [x] CI is currently green and this is ready for review
* [x] I am ready to test changes after they are applied and released

Signed-off-by: Toni Tauro <toni.tauro@adfinis.com>

### CODE REVIEW

File: charts/security-apps/README.md [hunk 1]
```diff
@@ -55,9 +55,9 @@ This chart is maintained by [Adfinis](https://adfinis.com/?pk_campaign=github&pk
 | gangway.chart | string | `"gangway"` | Chart |
 | gangway.destination.namespace | string | `"infra-gangway"` | Namespace |
 | gangway.enabled | bool | `false` | Enable gangway |
-| gangway.repoURL | string | [repo](https://charts.helm.sh/stable) | Repo URL |
-| gangway.targetRevision | string | `"0.4.*"` | [gangway Helm chart](https://github.com/helm/charts/tree/master/stable/gangway/) version |
-| gangway.values | object | [upstream values](https://github.com/helm/charts/tree/master/stable/gangway/values.yaml) | Helm values |
+| gangway.repoURL | string | [repo](https://gabibbo97.github.io/charts) | Repo URL |
```
Comment (vmaillot) [inline charts/security-apps/README.md:58]: i got a 4o4 on https://gabibbo97.github.io/charts/ is that a normal behaviour?

**Review by vmaillot — APPROVED** [review]

File: charts/security-apps/README.md (hunk 1 above)
Comment (eyenx) [reply charts/security-apps/README.md:58]: not normal, but it has not got a index.html the https://gabibbo97.github.io/charts/index.yaml is there to

Comment (vmaillot) [reply charts/security-apps/README.md:58]: alright, that's look good to me then :)

Comment (hairmare) [reply charts/security-apps/README.md:58]: yes normal, ours is fancy af ;)

**Review by hairmare — APPROVED** [review]
looks great! thanks for also adding examples which are always appreciated!

**Pull request merged**